community/cypher/spec-suite-tools/src/test/scala/cypher/feature/steps/SpecSuiteSteps.scala

```diff
@@ -73,6 +80,13 @@ trait SpecSuiteSteps extends FunSuiteLike with Matchers with TCKCucumberTemplate
     // do nothing, but necessary for the scala match
   }
 
+  private val PENDING = """^this scenario is pending on: (.+)$"""
+  And(PENDING) { (reason: String) =>
```

Comment:
I'm not sold on this pending thing. It allows one to write a whole lot of code that is not guaranteed to make it in. It's like writing `ignore` tests, isn't it?